MediaBrowser.Controller/Entities/BaseItem.cs

```diff
@@ -1327,8 +1327,9 @@ protected virtual BaseItem[] LoadExtras(List<FileSystemMetadata> fileSystemChild
                         }
 
                         // Use some hackery to get the extra type based on foldername
-                        Enum.TryParse(extraFolderName.Replace(" ", ""), true, out ExtraType extraType);
-                        item.ExtraType = extraType;
+                        item.ExtraType = Enum.TryParse(extraFolderName.Replace(" ", ""), true, out ExtraType extraType)
+                            ? extraType
+                            : Model.Entities.ExtraType.Unknown;
```

Comment:
Set the extra type as the new enum value `Unknown` instead of `null` when the type cannot be parsed